pkg/web/apps/apps_psql_repository_test.go

```diff
@@ -1,56 +1,241 @@
-// +build integration
-
 package apps
 
 import (
-	"reflect"
 	"testing"
 
-	"github.com/aerogear/mobile-security-service/pkg/config"
-	"github.com/aerogear/mobile-security-service/pkg/db"
-
 	"github.com/aerogear/mobile-security-service/pkg/models"
+	sqlmock "gopkg.in/DATA-DOG/go-sqlmock.v1"
 )
 
-func Test_appsPostgreSQLRepository_GetApps(t *testing.T) {
-	c := config.Get()
-	dbConn, err := db.Connect(c.DB.ConnectionString, c.DB.MaxConnections)
+func Test_appsPostgreSQLRepository_GetAppsWillReturnTwoApps(t *testing.T) {
```

Comment:
The mock data or func shows that is not working. See the CircleCI issue. 

<img width="1247" alt="screenshot 2019-02-15 at 17 12 11" src="https://user-images.githubusercontent.com/7708031/52872665-ff2a2500-3144-11e9-8cfa-601209caf13f.png">

Reply:
This is because the database which is created during integration tests does not have any seed data. This will need to be added. This is not directly caused by the mock data - it is separate but it will need to be added.

Reply:
This may be a tad out of scope of this task as it requires a whole new line of investigation.

Reply:
@camilamacedo86 I've added a function to seed the database before running the integration tests. This should succeed now.